stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -120,6 +121,12 @@ internal class PaymentSheetViewModel internal constructor(
             paymentIntent.value?.let { paymentIntent ->
                 StripeGooglePayLauncher(activity).startForResult(
                     StripeGooglePayLauncher.Args(
+                        environment = when (args.config?.googlePay?.environment) {
```

Comment:
this allows even `null` to be test - is that intentional?

Reply:
In practice, `args.config.googlePay` will always be non-null.